src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -1990,7 +1990,7 @@ internal string TimeSeparator
                     // Compute STIME from time format
                     _sTimeSeparator = GetTimeSeparator(longTimeFormat);
                 }
-                return _sTimeSeparator;
+                return _sTimeSeparator!;
```

Comment:
Maybe add a `Debug.Assert` on all these?

Reply:
Wouldn't it make the code even less readable, the condition would be almost identical to what is already in